WalletWasabi.Fluent/ViewModels/Wallets/Advanced/WalletCoins/WalletCoinsViewModel.cs

```diff
@@ -23,35 +22,99 @@ namespace WalletWasabi.Fluent.ViewModels.Wallets.Advanced.WalletCoins;
 [NavigationMetaData(Title = "Wallet Coins (UTXOs)")]
 public partial class WalletCoinsViewModel : RoutableViewModel
 {
-	private readonly WalletViewModel _walletViewModel;
 	private readonly IObservable<Unit> _balanceChanged;
-	private readonly ObservableCollectionExtended<WalletCoinViewModel> _coins;
-	private readonly SourceList<WalletCoinViewModel> _coinsSourceList = new();
-	[AutoNotify] private FlatTreeDataGridSource<WalletCoinViewModel>? _source;
+	private readonly WalletViewModel _walletViewModel;
+	private ObservableAsPropertyHelper<bool> _isAnySelected = ObservableAsPropertyHelper<bool>.Default();
+	[AutoNotify] private FlatTreeDataGridSource<WalletCoinViewModel> _source;
```

Comment:
Still null warning

Reply:
Sorry, I didn't get what warning was. I needed to double check :)

Reply:
`Non-nullable field '_source' is uninitialized. Consider declaring the field as nullable.`